src/gui/menu_filesystem.cpp

```diff
@@ -108,6 +108,8 @@ FileSystemMenu::refresh_items()
     item_id++;
   }
 
+  add_hl();
+  add_entry(-2, _("Open Directory"));
   add_hl();
```

Comment:
Which item has the id -1?

Reply:
Back

Reply:
I did some testing, and it seems like the back button does not trigger the `menu_action` function at all; however, looking at other files in the `src/supertux/menu` folder, it seems like the ID is overridden anyways, or that a non-standard back button is used.

I retract my comment, then.